zebra-state/src/service.rs

```diff
@@ -1554,6 +1557,60 @@ impl Service<ReadRequest> for ReadStateService {
                 .map(|join_result| join_result.expect("panic in ReadRequest::BestChainBlockHash"))
                 .boxed()
             }
+
+            // Used by get_block_template RPC.
+            #[cfg(feature = "getblocktemplate-rpcs")]
+            ReadRequest::ChainInfo() => {
+                metrics::counter!(
```

Comment:
A `req.count_metric()` call was recently added at the top of this fn that's already doing this.

Reply:
It seems i missed that change, thanks! I removed it at https://github.com/ZcashFoundation/zebra/pull/5659/commits/e80514a675a1c278b8c01d3b072ca68164d1fa0d . 

Also removed from `BestChainBlockHash`